libtcmu_log.h

```diff
@@ -33,6 +33,15 @@
 #define TCMU_LOG_DEBUG	LOG_DEBUG	/* debug-level messages */
 #define TCMU_LOG_DEBUG_SCSI_CMD	(LOG_DEBUG + 1)	/* scsi cmd debug-level messages */
 
+typedef enum {
+        TCMU_LOG_TO_STDOUT,
+        TCMU_LOG_TO_SYSLOG,
+        TCMU_LOG_TO_FILE,
+} tcmu_log_destination;
+
+typedef int (*log_output_fn) (int priority, const char *timestamp, const char *str, void *data);
+typedef void (*log_close_fn) (void *data);
```

Comment:
I am sorry about this, but between when I have you the original review comment and merging other patches, I guess we ended up adding a _t to the end of typedef definitions like this, so do

log_outpout_fn_t
log_close_fn_t